.github/workflows/manual-zcashd-deploy.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           --container-image electriccoinco/zcashd \
           --container-env ZCASHD_NETWORK="${{ inputs.network }}" \
           --machine-type ${{ vars.GCP_SMALL_MACHINE }} \
-          --network-interface=subnet=${{ vars.GCP_SUBNETWORK }} \
+          --subnet=${{ vars.GCP_SUBNETWORK }} \
```

Comment:
Nitpick/Optional: Would be nice to update this [here too](https://github.com/ZcashFoundation/zebra/blob/ext-address-nodes/.github/workflows/sub-deploy-integration-tests-gcp.yml#L202).